CRUD/forms.py

```diff
@@ -13,8 +18,16 @@ class Meta:
             'primary_email', 'secondary_email', 'primary_phone', 'secondary_phone',
             'special_conditions_and_comments', 'archived'
         )
+        widgets = {
```

Comment:
Use the materialize date picker instead: https://materializecss.com/pickers.html